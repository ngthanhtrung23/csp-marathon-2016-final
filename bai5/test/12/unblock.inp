.12223
.14443
005.63
77586.
9.58..
9AA8..
