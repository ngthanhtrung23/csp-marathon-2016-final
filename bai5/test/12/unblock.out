22
3 D 2
2 R 1
4 R 1
6 D 1
8 U 1
A R 3
5 D 1
8 D 1
0 R 3
5 U 3
7 R 1
9 U 4
7 L 1
5 D 3
0 L 3
5 U 1
8 U 1
A L 4
3 D 1
5 D 1
8 D 1
0 R 4

